### DISCUSSION TITLE
Search for similar issues while entering a new one

### DISCUSSION THREAD

**Discussion by SmartManoj** [community/community#23697] [New to GitHub]
<p>Is this feature removed or any third party have this?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/SmartManoj">@SmartManoj</a> :wave:</p>
<p>At one point this was in beta and as of early this year the engineering team decided that there were some concerns about how it worked, so it was turned off. The team is still thinking about how they can improve this experience for people, however we can’t say when or even if a specific feature for this will be reintroduced.</p>

**Comment by SmartManoj** [comment]
<p>Could give a little info about the concerns?</p>

**Comment by dandv** [comment]
I would really like to see this feature reinstated, because it helps popular discussion software drastically reduce the amount of duplicates. Prominent examples include StackOverflow and [Discourse](https://meta.discourse.org/t/more-options-for-your-topic-is-similar-to/80057).

**Comment by aaronvg** [comment]
Can we revive this? Adding a semantic search to see similar issues as you are typing a new one would reduce duplicates.
